radio/src/keys.cpp

```diff
@@ -427,10 +431,22 @@ bool keysPollingCycle()
   for (int i = 0; i < MAX_KEYS; i++) {
     event_t evt = keys[i].input(keys_input & (1 << i));
     if (evt) {
+      evt |= i;
+#if defined(KEYS_GPIO_REG_PAGEDN) && !defined(KEYS_GPIO_REG_PAGEUP)
```

Comment:
We should avoid using hardware specific definitions in the generic code. I believe this could be solved differently, maybe directly at driver level. Maybe we just need to have this defined as some kind of "hook point" that is defined in the HAL drivers.

Reply:
At least this isolates it to one place; but if there's a better way let me know.

Reply:
Right, maybe that's food for another PR. I've spotted a couple other places (not much left) that still use hardware key definitions in core / UI code.

For reference:
- `gui/common/stdlcd/radio_ghost_menu.cpp`
- `gui/common/stdlcd/view_telemetry.cpp`
- `gui/common/stdlcd/radio_sdmanager.cpp`
- `lua/interface.cpp`
- `lua/api_general.cpp`
- `main.cpp`
- `opentx.h`
- `simu.cpp`
- `thirdparty/libopenui/src/keyboard_number.cpp`
- `thirdparty/libopenui/src/keyboard_text.cpp`